packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -3015,6 +3032,13 @@ class TextSelectionGestureDetector extends StatefulWidget {
     required this.child,
   });
 
+  /// Called whenever [TapAndDragGestureRecognizer] has started tracking
```

Comment:
Nit: maybe we should reference `BaseTapAndDragGestureRecognizer` here instead.

Reply:
We should point out when exactly this happens. For example on every `PointerDownEvent`.